### DISCUSSION TITLE
Integration with Gemini

### DISCUSSION THREAD

**Discussion by daudln** [wagtail/wagtail-ai#58] [Q&A] [answered]
Hi Dev,

I think it be nicer if our repo (wagtail AI) shall support Gemini too.

I found this documentation helpfull for the integration

https://ai.google.dev/tutorials/python_quickstart

**Comment by tomusher** [comment, marked as answer]
Gemini should be supported already using the [`llm-gemini` plugin](https://github.com/simonw/llm-gemini). Let me know if you get a chance to try it out!
